types/gtag.js/gtag.js-tests.ts

```diff
@@ -158,3 +158,8 @@ gtag("consent", "default", {
     personalization_storage: "granted",
     security_storage: "denied",
```

Comment:
Hey, could you update new consent in this block instead of creating new consent block ?

```
gtag("consent", "default", {
    functionality_storage: "denied",
    personalization_storage: "granted",
    security_storage: "denied",
    ad_personalization: "denied",
    ad_user_data: "denied"
    }
```